src/modules/endpoint/DatabaseConnector.js

```diff
@@ -87,6 +98,10 @@ class DatabaseConnector {
   }
 
   getMetaCitiesPath (): string {
+    return `${CACHE_DIR_PATH}/metaCities.json`
```

Comment:
By following the current naming schema this should be:
```suggestion
    return `${CACHE_DIR_PATH}/cities-meta.json`
```